.editorconfig

```diff
@@ -0,0 +1,33 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+max_line_length = 120
+# visual setting for intelliJ IDE, no consequences on code validation
+# https://www.jetbrains.com/help/idea/editorconfig.html
+ij_visual_guides = 100, 120, 150
```

Comment:
this should probably live in ~/.editorconfig, and this file should probably not have the `root = true` in the beginning 🤔

Reply:
Nice point